code/modules/projectiles/boxes_magazines/internal/rifle.dm

```diff
@@ -28,3 +28,22 @@
 	max_ammo = 1
 	caliber = CALIBER_HARPOON
 	ammo_type = /obj/item/ammo_casing/harpoon
+
+/obj/item/ammo_box/magazine/internal/boltaction/rebarxbow/normal
+	max_ammo = 1
+	caliber = CALIBER_REBAR
+	ammo_type = /obj/item/ammo_casing/rebar
```

Comment:
Traditionally one would make `/obj/item/ammo_box/magazine/internal/boltaction/rebarxbow` the "normal" one, rather than having a "normal" subtype

Reply:
Similar situation to above, the rebarxbow/forced being a subtype of rebarxbow made the bow always have two shots regardless of being wrenched, due to it being compatable with both kinds of ammo at the same time

Reply:
Can't you null/reverse variables on the subtype?

Reply:
Gun code is _weird_, ammo and caliber are different, and I dont want to break something for all guns for a net gain of 7 lines of code

Reply:
I don't think you could do anything here which would break _all_ gun code.
Does it not work to have this type as simply `/obj/item/ammo_box/magazine/internal/boltaction/rebarxbow` ?
Would the premitted calibers not prevent the loading of the syndicate ammo?
What makes the upside of an ammo type apply even if is not in use?

Reply:
> I don't think you could do anything here which would break _all_ gun code. Does it not work to have this type as simply `/obj/item/ammo_box/magazine/internal/boltaction/rebarxbow` ? Would the premitted calibers not prevent the loading of the syndicate ammo? What makes the upside of an ammo type apply even if is not in use?

yes, the normal one is designed to not be able to fire the syndicate ammo, as it is a lot stronger than he base ammo. When setting a magazine type, any subset of those types would be able to be loaded, akin to how any shotgun shell fits in a shotgun magazine. 

Using/obj/item/ammo_box/magazine/internal/boltaction/rebarxbow/ would allow any crossbow to load syndicate ammunition, and because the NORMAL and FORCED calibers have the same ammo_type, it loads it as "forced" ammo, but since it was never wrenched it still acts as using the standard initial_caliber, meaning the misfire chance never applies.

Reply:
Ok so after some digging: the first bit with syndicate ammo is def due to subtyping, the wrenched thing wasn't. Still, it stops the normal one from firing syndicate ammo, which was the main point of having separate magazines.